VideoApp/VideoApp/SwiftUI/Screens/Room/RoomViewModel.swift

```diff
@@ -57,6 +74,15 @@ import Combine
             .sink { [weak self] error in self?.handleError(error) }
             .store(in: &subscriptions)
         
+        captionsManager.$error
+            .sink { [weak self] error in
+                guard let error = error else { return }
```

Comment:
If there's no error, do you want to clear `alertIdentifier`?

Reply:
Good question. If we cleared `alertIdentifier` it would allow for this:

1. Say there was a captions error and we display it to the user. Then the server recovers and clears the error. If the user had not already dismissed the error alert, clearing `alertIdentifier` would make it go away automatically. I'm thinking it's probably best UX to not automatically dismiss alerts since it could be confusing to the user.
2. Say there was a captions error and we display it to the user. The user dismisses the error which will cause `alertIdentifier` to be cleared. Later there is some other foo error displayed unrelated to captions. And then the captions error is resolved on the server, which clears `alertIdentifier`. The cleared captions error will cause the foo error to automatically dismiss if the user has not already dismissed it which would be a bug. I guess we could check to make sure we only clear it if it was a captions error. But that is getting a little elaborate.

Note that when the user dismisses the alert, the alert view clears `alertIdentifier`.

So after writing this ^ I'm thinking we just want to react to the positive case only, given we are using alerts for the UI.